Tests/SonarQube.TeamBuild.PreProcessor.Tests/TargetsInstallerTests.cs

```diff
@@ -75,22 +87,245 @@ public void InstallTargetsFile_Overwrite()
 
             CreateDummySourceTargetsFile(sourceTargetsContent1);
 
+            var msBuildPathSettings = new MsBuildPathSettings();
+
             InstallTargetsFileAndAssert(sourceTargetsContent1, expectCopy: true);
-            Assert.IsTrue(FileConstants.ImportBeforeDestinationDirectoryPaths.Count == 2, "Expecting two destination directories");
+            Assert.IsTrue(msBuildPathSettings.GetImportBeforePaths().Count() == 2, "Expecting two destination directories");
 
-            var path = Path.Combine(FileConstants.ImportBeforeDestinationDirectoryPaths[0], FileConstants.ImportBeforeTargetsName);
+            var path = Path.Combine(msBuildPathSettings.GetImportBeforePaths().First(), FileConstants.ImportBeforeTargetsName);
             File.Delete(path);
 
             CreateDummySourceTargetsFile(sourceTargetsContent2);
             InstallTargetsFileAndAssert(sourceTargetsContent2, expectCopy: true);
             InstallTargetsFileAndAssert(sourceTargetsContent2, expectCopy: true);
         }
 
+        [TestMethod]
```

Comment:
The new unit tests should provide 100% coverage, but I did not delete the existing integration tests.